_posts/10-03-01-Password-Hashing.md

```diff
@@ -8,17 +8,26 @@ anchor:  password_hashing
 Eventually everyone builds a PHP application that relies on user login. Usernames and passwords are stored in a
 database and later used to authenticate users upon login.
 
-It is important that you properly [_hash_][3] passwords before storing them. Password hashing is an irreversible,
-one-way function performed against the user's password. This produces a fixed-length string that cannot be feasibly
-reversed. This means you can compare a hash against another to determine if they both came from the same source string,
-but you cannot determine the original string. If passwords are not hashed and your database is accessed by an
-unauthorized third-party, all user accounts are now compromised. 
+It is important that you properly [_hash_][3] passwords before storing them. You should not _encrypt_ passwords.
```

Comment:
https://github.com/codeguy/php-the-right-way/pull/725/commits/3af83de93d4998709212644873ae267b1d9f42e7 -- I also changed a few things around to aid in clarity. What previously described "password hashing" is actually true of all cryptographic hash functions (including the ones not meant for passwords).